pulser/_seq_drawer.py

```diff
@@ -199,6 +202,43 @@ def phase_str(phi):
         a.set_ylabel(r'$\Omega$ (rad/µs)', fontsize=14, labelpad=10)
         b.set_ylabel(r'$\delta$ (rad/µs)', fontsize=14)
 
+        if draw_phase_area:
+            top = False  # Variable to track position of box, top or center.
+            draw_phase = any(
+                seq_.type.phase != 0 for seq_ in seq._schedule[ch]
+                if isinstance(seq_.type, Pulse)
+            )
+            for pulse_num, seq_ in enumerate(seq._schedule[ch]):
+                # Select only `Pulse` objects
+                if isinstance(seq_.type, Pulse):
+                    if sampling_rate:
+                        area_val = np.sum(
+                            np.abs(yaeff[int(seq_.ti):int(seq_.tf)])
+                        ) / 1e3 / np.pi
```

Comment:
What you need here is this:
```suggestion
                    if sampling_rate:
                        area_val = np.sum(cs_amp(
                            np.arange(seq_.ti, seq_.tf) * 1e-3
                        )) * 1e-3 / np.pi
```

Reply:
Test it with a low sampling rate

Reply:
Thanks again for this suggestion.

When I tried your suggestion, I get these values on the plot for a simulation:

![img1](https://user-images.githubusercontent.com/68844397/119649308-1be35600-be40-11eb-8894-4ecb0c0cd5ab.png)

If I try to use:

```diff
area_val = np.sum(cs_amp(
-                      np.arange(seq_.ti, seq_.tf) * 1e-3
                 )) * 1e-3 / np.pi
area_val = np.sum(cs_amp(
+                        np.arange(seq_.ti, seq_.tf)
                    )) * 1e-3 / np.pi
```
I get this plot:

![img2](https://user-images.githubusercontent.com/68844397/119649851-c0fe2e80-be40-11eb-806c-80181e721ac2.png)

Both the above plots are with a low sampling rate. Moreover, it gives such plausible values for different sampling rates (eg: 1, 0.1, 0.001).

It seems that the lower plot gives correct values? Could you let me know your thoughts?

(The above diff is not in the latest commit)

Reply:
Yes, you're clearly right. It seems that `cs_amp` is in ns instead of µs as I thought. Well spotted

Reply:
It seems to be fixed now..thanks for confirming!